src/features/mutation_minigame/nodes/dna/Nucleotide.gd

```diff
@@ -0,0 +1,65 @@
+extends PanelContainer
+
+@export_enum("A", "G", "T", "C", "BLANK") var base: String = "BLANK"
+signal mutation(node, mutation)
+@export var bg_visible: bool = true:
+	set = set_bg_visible
+		
+func set_bg_visible(value):
+	if value:
+		self.remove_theme_stylebox_override("Panel")
+	else:
+		self.add_theme_stylebox_override("Panel",StyleBoxEmpty.new())
+
+var nitro_bases :Dictionary = {
+	"A": {
+		"Tooltip": "Adenine",
+		"Color": Color.GREEN,
+		"Text": "A"
+	},
+	"G": {
+		"Tooltip": "Adenine",
+		"Color": Color.BLUE,
+		"Text": "G"
+	},
+	"T": {
+		"Tooltip": "Adenine",
+		"Color": Color.RED,
+		"Text": "T"
+	},
+	"C": {
+		"Tooltip": "Adenine",
+		"Color": Color.ORANGE,
+		"Text": "C"
+	},
+	
+}
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	var label = $Label
+	if base == "BLANK":
```

Comment:
Are we able to do something other than string comparisons? I.e., proper enum comparison?

Reply:
I'll work it into the global script.